src/nvim/ex_docmd.c

```diff
@@ -4184,9 +4184,6 @@ static char_u *invalid_range(exarg_T *eap)
         }
         break;
       case ADDR_UNSIGNED:
-        if (eap->line2 < 0) {
```

Comment:
hmm what does `ADDR_UNSIGNED` even mean if all ranges are unsigned? can the flag be removed completely?

Reply:
Are you asking me specifically or just in general? I have no damn clue but I can check around to figure out what it's supposed to be. The weird thing is that I can't seem to find it's ever assigned a value. It says in ex_cmds_defs.h that ADDR_UNSIGNED defaults to 1 but I can't find it.

Reply:
Ranges can be negative. `ADDR_UNSIGNED` prevents negative range for quickfix commands. Patch v8.1.1261, 35dc6d6e.